### PR TITLE
XW-5119 | New typography

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/x-wing

### CODE REVIEW

File: app/styles/typography/_typescale.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// These will live in design system after we ensure everything looks good in mobile-wiki
+.wds-text-2xs {
```
Comment (hakubo) [inline app/styles/typography/_typescale.scss]: wds-font-size-xxs {}

File: app/styles/typography/_typescale.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	font-size: 10px;
```
Comment (hakubo) [inline app/styles/typography/_typescale.scss]: hmm, maybe we should use rems here?
so if users zoom in, whole page zoom in?

File: app/styles/typography/_font-weight-scale.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// These will live in design system after we ensure everything looks good in mobile-wiki
+.wds-font-light {
```
Comment (hakubo) [inline app/styles/typography/_font-weight-scale.scss]: .wds-font-weight-light {}

File: app/styles/typography/_typescale.scss (hunk 2 above)
Comment (rybmat) [reply app/styles/typography/_typescale.scss]: mobile-wiki does not allow to zoom, do other apps allow?

File: app/styles/typography/_font-weight-scale.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+	font-weight: 300;
```
Comment (rybmat) [inline app/styles/typography/_font-weight-scale.scss]: we have $wds-fandom-font-weight-* variables in DS already that have the same values. Should we use them here?

File: app/styles/base/_typography.scss [hunk 5]
```diff
@@ -33,41 +38,58 @@ h3,
...
-	line-height: $wds-typescale-size-plus-1;
+	font-size: 16px;
+	font-weight: $wds-fandom-font-weight-medium;
 }
 
 h5,
 h6 {
-	font-size: $wds-typescale-size-plus-1;
-	line-height: $wds-typescale-size-plus-1;
+	font-size: 14px;
```
Comment (hakubo) [inline app/styles/base/_typography.scss]: should that extend font size placeholder from DS?

File: app/templates/components/widget-discussions.hbs [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {{#if posts}}
-	<div class="heading-wrapper">
-		<h2 class="heading">
+	<div class="heading-wrapper wds-font-size-xs">
```
Comment (hakubo) [inline app/templates/components/widget-discussions.hbs]: a browser would have less work to do if that was applied on a tag
possible?

File: app/styles/base/_typography.scss [hunk 7]
```diff
@@ -33,41 +38,58 @@ h3,
 h4,
 h5,
 h6 {
-	margin-bottom: 1.4rem;
-}
-
-ul,
-ol {
-	margin-left: 1.1rem;
+	margin-bottom: 18px;
```
Comment (hakubo) [inline app/styles/base/_typography.scss]: rems or px?

**Review by hakubo — APPROVED** [review]

**Pull request merged**